### ISSUE TITLE
Feature request: Option to 'lock' specific windows

### ISSUE THREAD

**Issue by TheJokingJack** [tabwrangler/tabwrangler#346]
I like to organize my tabs in windows, for example, one window is for studying, another for entertainment, and so forth. Sometimes, I keep multiple windows open, for example, to look at references for a project.
I'd like a feature where you can toggle Tab Lock on specific/the current window, so Tab Wrangler will ignore the tabs in that window.

**Comment by marijuanatech** [comment]
Good idea! There [has been discussion about it in the past](https://github.com/tabwrangler/tabwrangler/issues/13).  I'll play with the idea this coming week...

**Comment by KTamas** [comment]
@marijuanatech a PR on this would be very welcome.

**Comment by ssorallen** [comment]
Closing as duplicate of https://github.com/tabwrangler/tabwrangler/issues/13. This is a feature I have wanted for a while and plan to add next.
